src/App.css

```diff
@@ -2,6 +2,73 @@
     text-align: center;
 }
 
+/*
+Desktop Header
+ */
+.categories {
+    display: inline-block;
+    text-transform: uppercase;
+    color: black;
+    cursor: pointer;
+    height: 37px !important;
+}
+
+.categories > h4 {
+    margin: 10px 5px;
+    font-size: 12px;
+}
+
+.nav {
+    display: inline-block;
+    float: right;
+}
+
+.appBar {
+    background-color: black !important;
+    display: table !important;
+}
+
+#explore {
+    background-color: #8acc25;
+}
+
+#report {
+    background-color: #ffd700;
+}
+
+#resources {
+    background-color: #39c9bb;
+}
+
+.headerTitle {
+    display: inline;
+    font-size: 14px;
+    color: white;
+    position: absolute;
+    left: 27%;
+}
+
+@media (min-width: 960px) {
+    .headerTitle {
+        left: 24%;
+    }
+}
+
+@media (min-width: 1440px) {
+    .headerTitle {
+        left: 20%;
+    }
+}
+
+.logo {
```

Comment:
One mega-mega-mega nit is that I don't think this logo has a drop-shadow like the rest of the header does. IDK how tricky that is to make happen -- maybe something for the backlog?

Reply:
Just added a v. small box shadow for now.

Reply:
Also, we need a better asset for the logo - this is a screenshot 😆

Reply:
We also might want to throw an onClick event on the logo to nav back to home eventually but that's tbd.